app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -301,5 +301,24 @@ def authorize_destroy
       return deny_access unless @version.content_view.deletable?
       true
     end
+
+    # Handle sorting by 'version', which is composed of major and minor
+    def sort_by_version
+      sort_order = 'desc'
+
+      if params[:order]
+        sort_by, sort_order = params[:order].split(' ').map(&:downcase)
+        sort_order = sort_order == 'asc' ? sort_order : 'desc'
+      end
+
+      if params[:order].blank? || sort_by == 'version'
+        params.delete(:order)
```

Comment:
What's the purpose of deleting the order param?

Reply:
scoped_search helper sees it and tries to sort on the non-existent column resulting in an error

Reply:
Ah! Gotcha